e3db/types/search_range.py

```diff
@@ -1,38 +1,214 @@
 
 class Range():
     def __init__(self, key="CREATED", format="Unix", zone="UTC", zone_offset=None, before=None, after=None):
-        '''
-        before, after are naive datetime objects
+        """
+        Initialize the Range class for use in Search. This class can be manually created if needed, but 
+        the Search.range() method handles the Search workflow.
 
-        '''
-        self.__key= key
+        Parameters
+        ----------
+        key : str, optional
+            "CREATED|MODIFIED" (the default is "CREATED")
+
+        format : str, optional
+            Currently is not a supported parameter (the default is "Unix")
+
+        zone : str, optional
+            Since python time objects are naive or zone-agnostic, this will attempt to append
+            the proper timezone information to the time object for the query.
+            Currently supported are:
+                "PST":"-08:00", "MST":"-07:00", "CST":"-06:00", "EST":"-05:00", "UTC":"+00:00"
+                (the default is "UTC" if the proper timezone cannot be found, which represents +00:00)
+
+        zone_offset : str, optional
+            If provided this offset will be used over zone.
+            Accepts the format "[+|-]dd:dd"
+            (the default is None(UTC), which will attempt to use zone if provided)
+        
+        before : time, optional
+            Search only for records that come before this time 
+            (the default is None, which leaves no upper bound on the query)
+        
+        after : time, optional
+            Search only for record that come after this time 
+            (the default is None, which leaves no lower bound on the query)
+        
+        Returns
+        ----------
+        None
+        """
+        self.__key = key
         self.__format = format
         self.__zone = zone
         self.__before = before 
         self.__after = after
+        self.zone_dict = {
+            "PST":"-08:00",
+            "MST":"-07:00",
+            "CST":"-06:00",
+            "EST":"-05:00",
+            "UTC":"+00:00"
+        } 
         if zone_offset is not None:
             self.__zone_offset = zone_offset
         else:
-            zone_dict = {
-                "PST":"-08:00",
-                "MST":"-07:00",
-                "CST":"-06:00",
-                "EST":"-05:00",
-                "UTC":"+00:00"
-            } 
-            self.__zone_offset = zone_dict.get(zone, "+00:00")
+            self.__zone_offset = self.zone_dict.get(zone, "+00:00")
     
     @property
     def before(self):
+        """
+        Get the before time with time zone information appended as a string
+
+        Returns
+        -------
+        str
+            Before time properly formatted to send to search service.
```

Comment:
Same as other comment, remove Search Service in favor of either "server" or "E3DB"